vite.config.mjs

```diff
@@ -12,7 +14,7 @@ const commitHash = child
 export default defineConfig({
     plugins: [solidPlugin(), mkcert(), nodePolyfills()],
     server: {
-        https: true,
+        https: isNotDocker,
```

Comment:
Why no https via Docker?

Reply:
its better else you need to import the vite certs created inside the docker container or have cert errors. if you wont to expose it it better to run a proxy with ssl in front

Reply:
We are working under the assumption that we are running with HTTPS. Eg clipboard API does not work when you are on HTTP. We need HTTPS

Reply:
cclipboard works without htrtps, i think its just about the camera feature

Reply:
> Only supported on pages served over HTTPS.
https://stackoverflow.com/a/30810322

HTTPS and localhost